test/all_datatypes.eds

```diff
@@ -0,0 +1,310 @@
+[FileInfo]
+FileName=all.eds
+FileVersion=1
+FileRevision=1
+EDSVersion=4.0
+Description=OD that implements all available object types
```

Comment:
Not really accurate, it's all of the numeric *data* types, plus some compound types.

Reply:
yes, good point. Its missing `TIME_OF_DAY`,  `TIME_DIFFERENCE`, `PDO_COMMUNICATION_PARAMETER`, `PDO_MAPPING`, `SDO_PARAMETER`.

Reply:
My main concern was the "object types" vs. "data types" mixup.